test/views/layout.erb

```diff
@@ -22,7 +22,7 @@
     <%= script_tag jquery_src %>
     <script>
       // This is for test in override.js.
-      // Must go after jQuery is loaded, but before jquery-ujs.
+      // Must go before rails-ujs.
```

Comment:
I think that previous version was correct (except jquery-ujs) because jQuery is loaded in tests.